src/components/pages/photos/elements/aboutText/aboutText.js

```diff
@@ -0,0 +1,21 @@
+    this[roverNames.OPPORTUNITY] = 'Opportunity, also known as MER-B (Mars Exploration Rover – B) or MER-1, and nicknamed "Oppy", is a robotic rover that was active on Mars from 2004 until the middle of 2018. Launched on July 7, 2003, as part of NASA\'s Mars Exploration Rover program, it landed in Meridiani Planum on January 25, 2004, three weeks after its twin Spirit (MER-A) touched down on the other side of the planet. With a planned 90-sol duration of activity (slightly more than 90 Earth days), Spirit functioned until it got stuck in 2009 and ceased communications in 2010, while Opportunity was able to stay operational for 5111 sols after landing, maintaining its power and key systems through continual recharging of its batteries using solar power, and hibernating during events such as dust storms to save power. This careful operation allowed Opportunity to exceed its operating plan by 14 years, 46 days (in Earth time), 55 times its designed lifespan. By June 10, 2018, when it last contacted NASA, the rover had traveled a distance of 45.16 kilometers (28.06 miles).';
+    this[roverNames.SPIRIT] = 'Spirit, also known as MER-A (Mars Exploration Rover – A) or MER-2, is a robotic rover on Mars, active from 2004 to 2010. It was one of two rovers of NASA\'s Mars Exploration Rover Mission. It landed successfully within the impact crater Gusev on Mars at 04:35 Ground UTC on January 4, 2004, three weeks before its twin, Opportunity (MER-B), which landed on the other side of the planet. Its name was chosen through a NASA-sponsored student essay competition. The rover became stuck in a "sand trap" in late 2009 at an angle that hampered recharging of its batteries; its last communication with Earth was sent on March 22, 2010.';
+    this[cameraNames.CHEMCAM] = 'Chemistry and Camera Complex';
+    this[cameraNames.FHAZ] = 'Front Hazard Avoidance Camera';
+    this[cameraNames.MAHLI] = 'Mars Hand Lens Imager';
+    this[cameraNames.MARDI] = 'Mars Descent Imager';
+    this[cameraNames.MAST] = 'Mast Camera';
+    this[cameraNames.MINITES] = 'Miniature Thermal Emission Spectrometer (Mini-TES)';
+    this[cameraNames.NAVCAM] = 'Navigation Camera';
+    this[cameraNames.PANCAM] = 'Panoramic Camera';
+    this[cameraNames.RHAZ] = 'Rear Hazard Avoidance Camera';
+  }
+}
```

Comment:
I think this information might make more sense in an enum as it's just a load of constant strings that aren't going to change during execution. What do you reckon?